tests/integration/questionnaire/test_questionnaire_list_collector_content.py

```diff
@@ -0,0 +1,194 @@
+                "authorised-trader-eu-radio-repeating-block": "No",
+            }
+        )
+
+        self.post()
+
+        self.assertInBody("Give details about Company B")
+
+        self.assertInUrl("/companies-repeating-block-1/")
+
+        self.post(
+            {
+                "registration-number-repeating-block": "456",
+                "registration-date-repeating-block-day": "1",
+                "registration-date-repeating-block-month": "1",
+                "registration-date-repeating-block-year": "1990",
+            }
+        )
+
+        self.assertInBody("Give details about how Company B")
+
+        self.post(
+            {
+                "authorised-trader-uk-radio-repeating-block": "Yes",
+                "authorised-trader-eu-radio-repeating-block": "No",
+            }
+        )
+
+        self.assertInBody(
+            "You have previously reported the following companies. Press continue to updated registration and trading information."
+        )
+
+        self.post()
+
+        self.assertInUrl("questionnaire/sections/section-list-collector-contents/")
+
+        self.assertInBody("List Collector Contents")
+
+        self.post()
+
+        self.assertInUrl("/questionnaire/")
+
+    def test_hub_section_in_progress(self):
+        self.launchSurvey("test_list_collector_content_page")
+
+        self.post({"any-companies-or-branches-answer": "Yes"})
+
+        self.add_company("Company A", "123", "No")
+
+        self.post({"any-other-companies-or-branches-answer": "Yes"})
+
+        self.add_company("Company B", "456", "No")
+
+        self.post({"any-other-companies-or-branches-answer": "No"})
+
+        self.post()
+
+        self.post()
+
+        self.assertInUrl("/questionnaire/list-collector-content/")
+
+        self.assertInBody(
+            "You have previously reported the following companies. Press continue to updated registration and trading information."
+        )
+
+        self.post()
+
+        self.assertInUrl("/companies-repeating-block-1/")
+
+        self.assertInBody("Give details about Company A")
+
+        self.post(
+            {
+                "registration-number-repeating-block": "123",
+                "registration-date-repeating-block-day": "1",
+                "registration-date-repeating-block-month": "1",
+                "registration-date-repeating-block-year": "1990",
+            }
+        )
+
+        self.previous()
+
+        self.previous()
+
+        self.assertInBody("Partially completed")
+
+    def test_hub_section_in_progress_when_one_complete(self):
+        self.launchSurvey("test_list_collector_content_page")
+
+        self.post({"any-companies-or-branches-answer": "Yes"})
+
+        self.add_company("Company A", "123", "No")
+
+        self.post({"any-other-companies-or-branches-answer": "Yes"})
+
+        self.add_company("Company B", "456", "No")
+
+        self.post({"any-other-companies-or-branches-answer": "No"})
```

Comment:
Non issue - don't think we need the gaps every other line

Reply:
I am not sure what to do here, I just followed convention from other files like `tests/integration/questionnaire/test_questionnaire_list_collector.py`

Reply:
Ah ok 👍 looks like it varies, the `test_questionnaire_interstitial` one was what I'd been doing, could be one for team chat on what everyone finds easiest to read, or you can just resolve, not a big deal

Reply:
I don't we need gaps after every line, it looks very verbose. My preference is gaps after each logical chunk. Thoughts?

Reply:
I'd prefer that too

Reply:
So I grouped the journeys together, assertions together and method calls like "self.launchSurvey()" separate.